File: snippets/voice/group-call/group-call.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";
            
        string[] numbers = {"+1234567910", "+1234567911", "+1234567912"};

        TwilioClient.Init(accountSid, authToken);
        
        foreach (string number in numbers) {
                var call = CallResource.Create(
                    url: new Uri("http://demo.twilio.com/docs/voice.xml"), // Your TwiML Bin URL here.
                    to: new Twilio.Types.PhoneNumber(number),
                    from: new Twilio.Types.PhoneNumber("+15017122661") // Your Twilio phone number here.
                );

                Console.WriteLine(call.Sid);
                
        }
}

File: snippets/voice/group-call/group-call.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
// DANGER! This is insecure. See http://twil.io/secure
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$numbers = array("+1234567910", "+1234567911", "+1234567912");

foreach ($numbers as $number)
{
    $call = $twilio->calls
        ->create($number,
            "+15017122661", // Your Twilio phone number.
            array("url" => "http://demo.twilio.com/docs/voice.xml") // Your TwiML Bin URL here.
    );

    print($call->sid);
}
